akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/presentationpod/MakePresentationDownloadReqMsgHdlr.scala

```diff
@@ -146,7 +151,7 @@ trait MakePresentationDownloadReqMsgHdlr extends RightsManagementTrait {
       val exportJob: ExportJob = new ExportJob(jobId, JobTypes.DOWNLOAD, "annotated_slides", presId, presLocation, allPages, pagesRange, meetingId, "");
       val storeAnnotationPages: List[PresentationPageForExport] = getPresentationPagesForExport(pagesRange, pageCount, presId, currentPres, liveMeeting);
 
-      val isOriginalPresentationType = m.body.typeOfExport == "Original"
+      val isOriginalPresentationType = m.body.typeOfExport.contains("Original")
```

Comment:
It will not be clear for the future developers which is the possibilities here
I rather suggest to use `(m.body.typeOfExport == Original_Not-converted || m.body.typeOfExport == Original_Converted)`

Although it doesn't make sense for me the say it is the original converted.... if it is converted it is not the original anymore.
In this case I suggest to stick with the names `Original`,  `Converted` and `Annotated`!